worker/src/main/java/gov/cms/ab2d/worker/service/FileServiceImpl.java

```diff
@@ -34,4 +37,15 @@ public Path createDirectory(Path outputDir) {
         }
         return outputDirectory;
     }
+
+    @Override
+    public byte[] generateChecksum(File file) {
+        try (FileInputStream fileInputStream = new FileInputStream(file)) {
+            byte[] bytes = new SHA256.Digest().digest(fileInputStream.readAllBytes());
```

Comment:
We should definitely try and _stream_ the file into the digest function instead. `readAllBytes` will read the entire file into memory and our files are big.